tests/test_examples.py

```diff
@@ -0,0 +1,78 @@
+import itertools
+import sys
+from pathlib import Path
+from subprocess import PIPE  # noqa: S404
+from subprocess import Popen  # noqa: S404
+from subprocess import STDOUT  # noqa: S404
+from time import sleep
+from typing import List
+
+import pytest
+
+HERE = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve().parent
+
+
+# FIXTURES
+
+
+@pytest.fixture
+def path_1_simple() -> Path:
+    path = (HERE / ".." / "examples" / "1_simple").resolve()
```

Comment:
why not having fixtures for relevant folders inthe repo as we do in osparc. This way we can reuse it eaily for testing. For instance
```python
@pytest.fixture
def repo_folder() -> Path
    ...

@pytest.fixture
def docs_folder(repo_folder: Path) -> Path

@pytest.fixture
def tests_folder(repo_folder: Path) -> Path
    ...

@pytest.fixture
def examples_folder(repo_folder: Path) -> Path
    dirpath =  repo_folder / "examples"
    assert dirpath.exists()
    return dirpath


def test_something(examples_folder: Path)
    ...

@pytest.mark.parametrize("example_file", pytest.lazy_fixture('examples_folder').rglob("*.py"))
def test_something_else(example_file: Path)
    ...


```

Reply:
Will split and restructure the tests a bit